spikeinterface_gui/launcher.py

```diff
@@ -0,0 +1,425 @@
+from pathlib import Path
+import spikeinterface as si
+from spikeinterface.widgets.sorting_summary import _default_displayed_unit_properties
+
+from spikeinterface_gui.main import run_mainwindow
+from spikeinterface_gui.layout_presets import _presets
+
+
+class Launcher:
+    def __init__(self, analyzers_folder=None, backend="qt", verbose=False):
+        from spikeinterface_gui.main import check_folder_is_analyzer
+
+        if analyzers_folder is not None:
+            self.analyzer_folders = [
+                p for p in Path(analyzers_folder).iterdir() if p.is_dir() and check_folder_is_analyzer(p)
+            ]
+        else:
+            self.analyzer_folders = None
+
+        self.analyzer_path = None
+        self.recording_path = None
+        self.curation = True
+        self.displayed_properties = None
+        self.verbose = verbose
+
+        if backend == "qt":
+            self._qt_make_layout()
+        elif backend == "panel":
+            self._panel_make_layout()
+
+    def _load_analyzer(self):
+        if self.verbose:
+            print(f"Loading analyzer from {self.analyzer_path}...")
+        self.analyzer = si.load(self.analyzer_path, load_extensions=False)
```

Comment:
We should have a design from now to be able to open several windows and sevral analyzer at once.
We need a list or dict of this and relative windows.
Ctach the closing to detroy the analyzer later.
What do ytou think ?